.github/workflows/pipeline_v3_ci.yml

```
@@ -0,0 +1,262 @@
+          if [ $unit_failures -gt 0 ]; then
+            echo "Unit tests failed!"
+            exit 1
+          fi
+        fi
+
+  lint:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)